### PR TITLE
feat(ai.training): add `ID` column in job list

### PR DESCRIPTION
**Pull request by christophe-rannou**
Signed-off-by: Christophe Rannou <christophe.rannou@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix MLS-2184
| License          | BSD 3-Clause

## Description

Fix column name for job ID and add missing Name column

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html [hunk 1]
```diff
@@ -287,6 +287,10 @@
                 >
                     <thead>
                         <tr>
+                            <th class="oui-table__header">
+                                {{:: 'pci_projects_project_training_jobs_name' |
+                                translate}}
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html:292]: You can use `data-translate='pci_projects_project_training_jobs_name'` directly on `th` element.

File: packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html [hunk 2]
```diff
@@ -313,6 +317,9 @@
                             data-ng-repeat="job in $ctrl.runningJobs"
                             class="oui-table__row"
                         >
+                            <td class="oui-table__cell ellipsis">
+                                {{ job.spec.name }}
+                            </td>
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html:322]:
```suggestion
                            <td class="oui-table__cell ellipsis" data-ng-bind="job.spec.name">
                            </td>
```

File: packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html (hunk 1 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html]: done

File: packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html (hunk 2 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html]: done

**Review by JeremyDec — APPROVED** [review]

**Review by JeremyDec — CHANGES_REQUESTED** [review]
Can you please amend your commit to add internal ticket reference following the format:
`ref: MLS-2184`?

**Comment by christophe-rannou** [comment]
@JeremyDec Is the commit message ok for you ?

**Comment by antleblanc** [comment]
Hi @christophe-rannou!

`ref: MLS-2184` can be located into the body commit message rather than on the summary.

This is definitely something that we can adjust during the **Squash and Merge** part.

My only concern is maybe around the commit itself, you're using commit type `feat`
but the summary is saying `fix job list column names`.

IMHO and according to the existing diff, you are introducting a `feat` (feature) so maybe
there's no need to specify `fix` here.

File: packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html [hunk 3]
```diff
@@ -277,7 +277,7 @@
                     data-ng-if="$ctrl.runningJobs.length == 0"
                 >
                     <span
-                        data-translate="{{:: 'pci_projects_project_training_dashboard_jobs_list_none' | translate  }}"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html:280]: Good catch!

**Review by antleblanc — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**